src/test/java/uk/gov/companieshouse/api/strikeoffobjections/chips/ChipsClientTest.java

```diff
@@ -35,16 +48,25 @@
     private RestTemplate restTemplate;
 
     @Test
-    void testSendMessageToChips(){
+    void testSendMessageToChips() {
+        Utils.setTestAttachmentsWithLinks(ATTACHMENTS);
         ReflectionTestUtils.setField(chipsClient, "chipsRestUrl", CHIPS_REST_URL);
         ChipsRequest chipsRequest = new ChipsRequest(
                 REQUEST_ID,
-                COMPANY_NUMBER
+                COMPANY_NUMBER,
+                ATTACHMENTS,
+                CUSTOMER_EMAIL,
+                REASON
         );
 
+        assertEquals("link to SELF 1",
```

Comment:
this looks like it should go in a different test, the testSendMessageToChips() is testing the sendToChips method. It looks like the assert on L62 and L64 is being made before the method being tested is called. 
The asserts are testing that the ChipsRequest constructor to check it is parsing the List of attachments correctly, so I would say these parts belong in their own test for the ChipsRequest class. maybe a testAttachmentConstruction in ChipsRequestTest class.